WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.xib

```diff
@@ -83,34 +85,28 @@
                                     <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                                 </view>
                                 <view contentMode="scaleToFill" verticalHuggingPriority="1000" translatesAutoresizingMaskIntoConstraints="NO" id="10Z-y1-ZQ6" userLabel="Action Background View">
-                                    <rect key="frame" x="0.0" y="337" width="248" height="125"/>
+                                    <rect key="frame" x="0.0" y="337" width="248" height="115"/>
                                     <subviews>
                                         <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="6" translatesAutoresizingMaskIntoConstraints="NO" id="o73-de-cj4">
-                                            <rect key="frame" x="0.0" y="0.0" width="248" height="125"/>
+                                            <rect key="frame" x="0.0" y="0.0" width="248" height="115"/>
                                             <subviews>
                                                 <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="15" translatesAutoresizingMaskIntoConstraints="NO" id="mhI-fa-Yzw">
-                                                    <rect key="frame" x="0.0" y="0.0" width="248" height="95"/>
+                                                    <rect key="frame" x="0.0" y="0.0" width="248" height="79"/>
                                                     <subviews>
                                                         <button opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="1000" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="ZHa-is-GJK" userLabel="secondary action button">
-                                                            <rect key="frame" x="0.0" y="0.0" width="248" height="40"/>
-                                                            <constraints>
-                                                                <constraint firstAttribute="height" constant="40" id="Gcd-Af-CS9"/>
-                                                            </constraints>
+                                                            <rect key="frame" x="0.0" y="0.0" width="248" height="32"/>
                                                             <fontDescription key="fontDescription" type="system" weight="semibold" pointSize="16"/>
-                                                            <state key="normal" title="Primary Action Button">
+                                                            <state key="normal" title="Print">
```

Comment:
nit: the text in the xib doesn't really matter since we always set new text to the button title, but since this modal xib is used for all kinds of card-present payment alerts maybe we can keep the placeholder button titles to be generic?

Reply:
Done 3c3f9276aeb67398da8f30134d8f81a491cf547e